AzurePipelinesTemplates/win32metadata-onebranch.yml

```diff
@@ -0,0 +1,200 @@
+parameters:
+- name: "PipelineType"
+  type: string
+  default: "PullRequest"
+- name: "RepoDirectory"
+  type: string
+  default: "win32metadata"
+- name: OfficialBuild
+  type: boolean
+  default: false
+
+stages:
+- stage: scrape
+  displayName: "Scrape"
+  jobs:
+  - job: scrape
+    strategy:
+      matrix:
+        x86:
+          arch: 'x86'
+          generateMetadataArgs: '-scrapeConstants'
+        x64:
+          arch: 'x64'
+        arm64:
+          arch: 'arm64'
+    displayName: "Scrape headers"
+    timeoutInMinutes: 60
+    variables:
+      ob_outputDirectory: '${{parameters.RepoDirectory}}\generation\WinSDK\obj'
+      ob_artifactBaseName: 'generated'
+      ob_artifactSuffix: '_$(arch)'
+      ob_sdl_binskim_break: true # https://aka.ms/obpipelines/sdl
+    pool:
+      type: windows
+    steps:
+    - template: win32metadata-checkout.yml
+      parameters:
+        RepoDirectory: ${{ parameters.RepoDirectory }}
+
+    - task: UseDotNet@2
+      displayName: ⚙ Install .NET SDK
+      inputs:
+        packageType: sdk
+        useGlobalJson: true
+
+    # TODO - this no longer sets an Azure Devops pipeline build number, OneBranch has a different way of doing that

```

Comment:
Is this TODO still pending?

Reply:
It is, this Azure Devops pipeline name doesn't get properly set right now

Reply:
The current release pipeline uses the pipeline build number so I don't think we can merge this until this is resolved.